internal/component/database_observability/mysql/collector/parser/parser_tidb_test.go

```diff
@@ -146,3 +147,51 @@ func TestParserTiDB_ExtractTableNames(t *testing.T) {
 		})
 	}
 }
+
+func TestParserTiDB_CleanTruncatedText(t *testing.T) {
+	testcases := []struct {
+		name    string
+		sql     string
+		want    string
+		wantErr bool
```

Comment:
We should assert on the error containing the expected info/type too. 
Instead of wantErr being a `bool`, it could be an `error`